rpcs3/rpcs3qt/memory_viewer_panel.cpp

```diff
@@ -62,18 +71,18 @@ memory_viewer_panel::memory_viewer_panel(QWidget* parent, u32 addr, std::functio
 	m_fontMetrics = new QFontMetrics(mono);
 
 	// Layout:
-	QVBoxLayout* vbox_panel = new QVBoxLayout();
+	QVBoxLayout* vbox_panel = new QVBoxLayout(this);
```

Comment:
all of these parent additions are kinda obsolete, as the widgets are passed to layouts anyway.
"Note: The ownership of item is transferred to the layout, and it's the layout's responsibility to delete it."